jumpstart-week2/README.md

```diff
@@ -0,0 +1,222 @@
+# Frontend Basics and Introduction to git
+
+Terminal cont.
+Git basics
+  - idea of version control
+  - local vs remote repos
+  - commit, push, pull
+Front end basics
+- HTML
+    - elements
+    - tags
+    - attributes
+- Basic CSS
+- Javascript
+
+
+Basic plan
+1. Have everyone clone the git repository
+- Open terminal
+- Go to a folder
+- Run a git clone
+
+2. Tell everyone to ignore most of the boiler plate, get everyone to be able to run our base code
+- Introduce the idea of node
+- npm install
+- npm run start
+- Open localhost, explain the idea
+
+3. Use our example code to lecture about the basics of HTML
+- Bring up file for the homepage
+- Talk about format of HTML
+- Introduce tags, attributes
+- divs & spans
```

Comment:
Capitalize

Reply:
I believe these should stay uncapitalized

Reply:
ok sure